pom.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>io.jenkins.infra</groupId>
+    <artifactId>pipeline-library</artifactId>
+    <version>0.0.1</version>
+    <name>Jenkins Pipeline Shared Library</name>
+    <description>Pipeline Shared Library containing utility steps.</description>
+    <url>https://github.com/jenkins-infra/pipeline-library</url>
+
+    <licenses>
+        <license>
+            <name>MIT License</name>
+            <url>http://opensource.org/licenses/MIT</url>
+        </license>
+    </licenses>
+
+    <scm>
+        <connection>scm:git:git://github.com/jenkins-infra/pipeline-library.git</connection>
+        <developerConnection>scm:git:ssh://git@github.com/jenkins-infra/pipeline-library.git</developerConnection>
+    </scm>
+
+    <properties>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.codehaus.groovy</groupId>
+            <artifactId>groovy-all</artifactId>
+            <version>2.4.3</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.12</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.lesfurets</groupId>
+            <artifactId>jenkins-pipeline-unit</artifactId>
+            <version>1.1</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.yaml</groupId>
+            <artifactId>snakeyaml</artifactId>
+            <version>1.21</version>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
+    <build>
+        <scriptSourceDirectory>vars</scriptSourceDirectory>
+        <testSourceDirectory>src</testSourceDirectory>
+        <resources>
+            <resource>
+                <directory>resources</directory>
+            </resource>
+        </resources>
+        <plugins>
+            <plugin>
+                <groupId>org.codehaus.groovy</groupId>
+                <artifactId>groovy-eclipse-compiler</artifactId>
+                <version>2.9.2-01</version>
```

Comment:
BTW there were some recent changes to the archetype here.

Reply:
After picking up the latest changes:
- https://github.com/jenkins-infra/pipeline-library/pull/93/commits/875e54bbcff05a23475441721ea5944f48a6df11

The test goal didn't execute anything and I managed to find implicit version changed of the surefire.

So I had to enforce the version as the previous one:
- https://github.com/jenkins-infra/pipeline-library/pull/93/commits/ba1396adea9a074006a411c25f6523d519d08c36

Not sure whether that's the expected behaviour in the archetype but with the latest commit everything seems to be back to business.